### DISCUSSION TITLE
Hex codes for shap summary plot

### DISCUSSION THREAD

**Discussion by mhanauer** [shap/shap#2381] [General]
Sorry silly question, but I love the colors for the shap summary plots (blue and red): https://shap.readthedocs.io/en/latest/example_notebooks/overviews/An%20introduction%20to%20explainable%20AI%20with%20Shapley%20values.html

I am wondering what the hex codes are?

Thanks!

**Comment by rlcauvin** [comment]
The [ColorZilla](https://www.colorzilla.com/chrome) Chrome extension (or Firefox plugin) is really useful for getting the RGB and other hex codes. You just click the pixel for which you want to know the hex codes, and it copies the RGB value to the clipboard.

Here's what ColorZilla told me about the blue and red in the SHAP plots:

blue: #008BFB
red: #FCFCFC
